tests/pipeline/test_us_equity_pricing_loader.py

```diff
@@ -395,6 +396,40 @@ def test_load_adjustments_from_sqlite(self):
                 self.assertEqual(adj.last_col, expected.last_col)
                 assert_allclose(adj.value, expected.value)
 
+    def test_load_adjustments_to_df(self):
+        reader = self.adjustment_reader
+        adjustment_dfs = reader.unpack_db_to_component_dfs()
+
+        name_and_raw = (
+            ('splits', SPLITS),
+            ('mergers', MERGERS),
+            ('dividends', DIVIDENDS_EXPECTED)
+        )
+
+        def create_expected_table(df, name):
+            expected_df = df.copy()
+
+            for colname in reader._datetime_int_cols[name]:
+                expected_df[colname] = expected_df[colname].astype(
+                    'datetime64[s]'
+                )
```

Comment:
Not sure of the context where `unpack_db_to_component_dfs` is being used, but it this conversion something we could do within the method before returning?